dynamiqs/integrators/apis/jssesolve.py

```diff
@@ -28,18 +28,20 @@ def jssesolve(
         This function has not been implemented yet. The following API is indicative
         of the planned implementation.
 
+    In quantum optics, the jump SSE describes the evolution of a quantum system under
+    photon counting schemes, with unit detection efficiency and zero dark count rate.
     This function computes the evolution of the state vector $\ket{\psi(t)}$ at time $t$,
-    starting from an initial state $\ket{\psi_0}$, according to the jump SSE (with
-    $\hbar=1$ and where time is implicit(1))
+    starting from an initial state $\ket{\psi_0}$, according to the jump SSE ($\hbar=1$,
+    time is implicit(1))
     $$
         \begin{split}
             \dd\ket\psi = \Bigg[
                 &-iH \dt
                 - \frac12 \sum_{k=1}^N \left(
-                    L_k^\dag L_k - \braket{\psi| L_k^\dag L_k |\psi}
+                    L_k^\dag L_k - \braket{L_k^\dag L_k}
                 \right) \dt \\\\
                 &+ \sum_{k=1}^N \left(
-                    \frac{L_k}{\sqrt{\braket{\psi| L_k^\dag L_k |\psi}}} - I
+                    \frac{L_k}{\sqrt{\braket{L_k^\dag L_k}}} - 1
```

Comment:
The `I` is important.

Reply:
Why is it important? It's not defined anywhere, plus it is standard notation to replace identity with a `1`.